index.ts

```diff
@@ -111,16 +112,24 @@ const fetchOmnivore = async (
       for (const { title, author, slug, description } of articles) {
         const { labels, highlights, savedAt } = await loadArticle(slug, apiKey)
 
-        const content = `[${title}](https://omnivore.app/me/${slug})
-        collapsed:: true
-        author:: "${author}"
-        labels:: ${
-          labels
-            ? labels.map((l: { name: string }) => `[[${l.name}]]`).join(' ')
-            : 'null'
+        // Build content string
+        let content = `[${title}](https://omnivore.app/me/${slug})`
+        content += '\ncollapsed:: true'
+        if (author) {
+          content += `\nauthor:: ${author}`
+        }
+
+        const joinedLabels = (labels ?? []).map((l: Label) => l.name).join()
```

Comment:
We should not remove `[[]]` from the label name as this would remove the internal reference

Reply:
Sorry just noticed this @satindar

Reply:
> We should not remove `[[]]` from the label name as this would remove the internal reference

Oh I thought this was some sort of nested array for joining. Worked without then so I thought it wasn't needed. Now it makes more sense! I'll add them back. Thanks for noticing.